src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -204,29 +230,23 @@ export const NavBar: React.FC<NavBarPropsType> = ({ hasBackButton = true }) => {
   const showSoundCloudRadio =
     (currentVenue?.showRadio && isSoundCloud) ?? false;
 
+  const showRadio = showNormalRadio || showSoundCloudRadio;
+  const volumeIcon = !volume ? faVolumeMute : faVolumeUp;
+
   return (
     <>
       <header>
         <div className={`navbar navbar_playa ${!isOnPlaya && "nonplaya"}`}>
           <div className="navbar-container">
             <div className="nav-logos">
-              <div className="nav-sparkle-logo">
-                <div />
-              </div>
-              <div
-                className="nav-sparkle-logo_small"
-                onClick={navigateToHomepage}
-              >
-                <div />
-              </div>
               {shouldShowHomeButton && (
                 <FontAwesomeIcon
-                  icon={faHome}
-                  className="NavBar__home-icon"
+                  icon={faCaretLeft}
+                  className="NavBar__home--icon"
                   onClick={navigateToHomepage}
                 />
               )}
-
+              <div className="nav-sparkle-logo" onClick={navigateToHomepage} />
```

Comment:
Good opportunity to add `NavBar__sparkle-logo` class alongside or instead of `nav-sparkle-logo` for future use

Reply:
fixed 61d9df7